addons/web/static/src/search/search_model.js

```diff
@@ -827,8 +827,20 @@ export class SearchModel extends EventBus {
 
         const tree = treeFromDomain(domain, { distributeNot: !this.isDebugMode });
         const trees = !tree.negate && tree.value === "&" ? tree.children : [tree];
+
+        const getDescription = async (model, tree) => {
+            const description = [];
+            description.push(await this.getDomainTreeDescription(model, tree));
+            if (isTree(tree.value)) {
```

Comment:
That part should be integretated to getDomainTreeDescription
It is missing in domain_field.js

Reply:
that part does not work for a or with any inside

Reply:
yup. I modified it